build/RunTestsOnHelix.cmd

```diff
@@ -28,7 +28,15 @@ robocopy %HELIX_CORRELATION_PAYLOAD%\t\TestExecutionDirectoryFiles %TestExecutio
 REM call dotnet new so the first run message doesn't interfere with the first test
 dotnet new --debug:ephemeral-hive
 REM avoid potetial concurrency issues when nuget is creating nuget.config
-dotnet nuget list source
+dotnet nuget list source --configfile %TestExecutionDirectory%\nuget.config
 REM We downloaded a special zip of files to the .nuget folder so add that as a source
 dotnet new nugetconfig
-dotnet nuget add source %DOTNET_ROOT%\.nuget --configfile nuget.config
+dotnet nuget add source %DOTNET_ROOT%\.nuget --configfile %TestExecutionDirectory%\nuget.config
+dotnet nuget remove source dotnet6-transport --configfile %TestExecutionDirectory%\nuget.config
```

Comment:
You can also add:
- dotnet6-internal-transport
- dotnet7-internal-transport

Reply:
I don't see those in any public or internal nuget.config. Where are those coming from? The internal PRs are still going to have a bunch of darc-int feeds which won't help here.

I was wondering if I could also remove the dotnet-tools and dotnet-libraries ones. Any idea? I guess I could try it and see what happens.

Reply:
Those are added dynamically via SetupNugetSources.sh/ps1, based on the feeds that appear in the original nuget.config file. It's how we transparently handle internal builds.

Reply:
the darc-int feeds do need to be there, since they are filled with shipping packages from the current build.

Reply:
Ahh, hadn't realized they were dynamically added. I'm now removing them as well. Let me know if this works for you and I'll cherry-pick to 7.0.2xx so it can go in now and create a 6.0.1xx version of this PR.